visidata/aggregators.py

```diff
@@ -146,17 +182,17 @@ def percentile(pct, helpstr=''):
 def quantiles(q, helpstr):
     return [percentile(round(100*i/q), helpstr) for i in range(1, q)]
 
-vd.aggregator('min', min, 'minimum value')
-vd.aggregator('max', max, 'maximum value')
-vd.aggregator('avg', mean, 'arithmetic mean of values', type=float)
-vd.aggregator('mean', mean, 'arithmetic mean of values', type=float)
-vd.aggregator('median', statistics.median, 'median of values')
+vd.aggregator('min', min, 'minimum value', type=anytype)
```

Comment:
Why are min/max forcibly set to `anytype` instead of letting them take the type of their source column?

I understand why avg/mean/median want to be more generic; maybe this is where we have both an 'input type' which remains `float` for these, and a 'result type' which is more variable.